electron/js/menu/tray.js

```diff
@@ -70,7 +70,9 @@ module.exports = {
         this.useDefaultIcon();
       }
       win.setOverlayIcon(count && process.platform === 'win32' ? iconOverlayPath : null, locale.getText('unreadMessages'));
-      win.flashFrame(!win.isFocused() && count > lastUnreadCount);
+      if (process.platform === 'win32') {
```

Comment:
no no no @lipis, it is very important to have it on Linux, please keep!

some Desktop Environments do nothing about it, but some do!